sdc/datatypes/common_functions.py

```diff
@@ -131,3 +132,19 @@ def _append_list_string_array_impl(A, B):
                 return new_data
 
             return _append_list_string_array_impl
+
+
+@numba.njit
+def _hpat_ensure_array_capacity(new_size, arr):
+    '''Function creating a copy of numpy array with a size more than specified'''
```

Comment:
Could you apply below recommendation everywhere in this PR?
[pep8 recommendation](https://www.python.org/dev/peps/pep-0008/#string-quotes): For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257.

Reply:
Applied